drivers/hmis_csv_twenty_twenty_four/spec/models/exporter/single_enrollment_tests.rb

```diff
@@ -25,6 +25,12 @@
       csv = CSV.read(csv_file_path(@enrollment_class), headers: true)
       expect(csv.first['PersonalID']).to_not be_empty
     end
+
+    it 'Correctly limits the length of PreferredLanguageDifferent to 100 characters' do
+      expect(@exporter.enrollment_scope.first.PreferredLanguageDifferent.length).to eq(200)
+      csv = CSV.read(csv_file_path(@enrollment_class), headers: true)
+      expect(csv.first['PreferredLanguageDifferent'].length).to eq(100)
```

Comment:
Something else is masking this, I commented out `enforce_lengths`, and the test still passes.

Reply:
Found and fixed.  Thank you